wiki/Tournaments/SS_Tournament/2018/en.md

```diff
@@ -0,0 +1,286 @@
+# SS Tournament 2018
+
+![SS Tournament 2018](Banner.jpg)
+
+SS Tournament 2018 is a osu!standard tournament organized and ran by ![][flag_CA] [Sparhten](https://osu.ppy.sh/users/7601720), ![][flag_US] [NeonLights](https://osu.ppy.sh/users/5277199), and ![][flag_GR] [FrozenPheonix](https://osu.ppy.sh/users/9696149).
+It is a open rank 3v3 tournament with rosters of 3 - 4 players held semi annually.
```

Comment:
Is this paragraph supposed to be by itself, or be part of the one above it?
If by itself, add another blank line before this one. If it's supposed to be part of the one above it, put these two other.